FhirToDataLake/src/Microsoft.Health.AnalyticsConnector.Common/Configurations/DicomServerConfiguration.cs

```diff
@@ -17,5 +17,10 @@ public class DicomServerConfiguration
 
         [JsonProperty("authentication")]
         public AuthenticationType Authentication { get; set; } = AuthenticationType.None;
+
+        // Azure DICOM server default audience is "https://dicom.healthcareapis.azure.com".
+        // Refer https://learn.microsoft.com/en-us/azure/healthcare-apis/dicom/dicom-get-access-token-azure-cli-old#obtain-a-token
+        [JsonProperty("audience")]
+        public string Audience { get; set; } = "https://dicom.healthcareapis.azure.com";
```

Comment:
need we provide the audience field in appsetting.json?

Reply:
Added audience in **appsettings-example-v2.json**